cli/cmd/install.go

```diff
@@ -397,7 +397,7 @@ func render(w io.Writer, values *l5dcharts.Values, stage string, options valuesp
 	}
 
 	if stage == "" || stage == controlPlaneStage {
-		overrides, err := renderOverrides(values, values.GetGlobal().Namespace)
+		overrides, err := renderOverrides(vals, vals["global"].(map[string]interface{})["namespace"].(string))
```

Comment:
I think these casts open us up to some pretty ugly crashes.  Since `renderOverrides` converts vals into a `Tree` anyway, what do you think of this:

Remove the namespace parameter to `renderOverrides`.  In `tree.go` add a safe `func (t Tree) getString(path []string) (string, error)` which traverses a path into the tree and attempts to return the value stored there, or returns an error if that path does not exist or the value is not a string.  Then from within `renderOverrides` we can find the namespace by calling `valuesTree.getString([]string{"global", "namespace"})`.

I actually had written this `getString` function in the past, but it wasn't used anywhere so I deleted it.

Reply:
That seems like a great idea. Added it along with unit tests.